### ISSUE TITLE
[Feature]: Release notes for F-Droid

### ISSUE THREAD

**Issue by balporig** [simplex-chat/simplex-chat#3723]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android

### Feature

Add Release notes/changelog to both the public and simplex hosted F-Droid repos

**Comment by Martina-Neumayer** [comment]
Yes I will be very nice to have those informations, especially about the minimal required android version. Because it is very confusing and annoying to try to update the app to the latest version and not being able to due to the android 8.0 on the device.

Btw.. That's suxx completely that the v8 is not longer supported. Lots of people are still using it and not have any option to upgrade their systems.
It is really so hard to add an release for this particular one system version too?
